src/main/webapp/src/components/entityAnalysisInfo.js

```diff
@@ -82,44 +82,45 @@ class EntityAnalysisInfo extends React.Component {
       return <p>No entities were found.</p>;
     }
 
-    /**
-     * Function will draw a pie chart with static data representing the most important words in
-     * our song's context.
-     */
-    const drawTop10WordsChart = (element) => {
-      const data = new google.visualization.DataTable();
-      data.addColumn("string", "Name");
-      data.addColumn("number", "Salience");
-      this.state.entityAnalysisInfo.forEach((entity) => {
-        data.addRow([entity.name, entity.salience]);
-      });
-
-      const options = {
-        title: "Most important words",
-        width: 600,
-        height: 500,
-      };
-
-      const entityChart = new google.visualization.PieChart(element);
-      entityChart.draw(data, options);
-    }
-
-    const loadChartAPI = (element) => {
-      // Load the Visualization API and the corechart package.
-      google.charts.load("current", { packages: ["corechart"] });
-
-      // Set a callback to run when the Google Visualization API is loaded.
-      google.charts.setOnLoadCallback(function () {
-        drawTop10WordsChart(element);
-      });
-    }
-
+    /** Gather the most important words and their data in simple arrays. */
+    var top10WordsData = [];
+    var top10WordsLinks = [];
+    top10WordsData.push(['Word and Type', 'Importance'])
+    this.state.entityAnalysisInfo.forEach((entity) => {
+      top10WordsData.push([entity.name + ' (' + entity.type + ')', entity.salience]);
```

Comment:
Looks like you hard code brackets. I think if you want to change something in data that you send you couldn't do it.

Reply:
The brackets are purely cosmetic, I don't think they conflict with any of the data. Can you tell me more about your concern?